hawkular-inventory-api/src/test/java/org/hawkular/inventory/api/test/AbstractBaseInventoryPersistenceCheck.java

```diff
@@ -2066,4 +2091,9 @@ void runObserverTest(Class<T> entityClass, int nofCreatedRelationships, int nofD
     private interface TetraFunction<T, U, V, W, R> {
         R apply(T t, U u, V v, W w);
     }
+
+    private <T> List<T> toList(Page<T> page) {
```

Comment:
What about having this directly as an instance method of `Page`? With some big fat warning about perf consequences...